### PR TITLE
Add support for credentials from the Jenkins credentials store

### PR DESCRIPTION
**Pull request by maxvodo**
By default we using username/password from original text fields (config.jelly).
If credentials ID provided in plugin settings - using them for BitBucket API auth.

Code from #64 reviewed and integrated into last master.
Resolve issue #33 and integrate #64 pull request.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -8,7 +8,7 @@
 
   <artifactId>bitbucket-pullrequest-builder</artifactId>
   <name>Bitbucket Pullrequest Builder Plugin</name>
-  <version>1.4.14-SNAPSHOT</version>
+  <version>1.4.15</version>
```
Comment (nishio-dens) [inline pom.xml]: This line is using for my release script(maven release plugin), so please do not change it.

**Comment by praveen12bnitt** [comment]
Now that it retains the backward compatibility, can we expect this to be merged to master anytime soon?
@maxvodo  thanks for doing this.

File: pom.xml (hunk 1 above)
Comment (maxvodo) [inline pom.xml]: Ok, sure!

**Comment by maxvodo** [comment]
@nishio-dens, after last commits pr looks good for me :ok_hand:

**Comment by nishio-dens** [comment]
It seems good, thanks.

**Comment by tnguyen14** [comment]
Should there be an update to the documentation for this new feature?

**Pull request merged**